mypyc/codegen/literals.py

```diff
@@ -3,15 +3,24 @@
 from typing import Final, Union
 from typing_extensions import TypeGuard
 
-# Supported Python literal types. All tuple / frozenset items must have supported
+# Supported Python literal types. All tuple / frozenset / dict items must have supported
 # literal types as well, but we can't represent the type precisely.
```

Comment:
is there a reason we can't represent the type precisely? I think with a HashableLiteralValue defined before LiteralValue, we can do it pretty cleanly